homeassistant/components/dsmr/sensor.py

```diff
@@ -387,6 +423,113 @@ class MbusDeviceType(IntEnum):
         device_class=SensorDeviceClass.GAS,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    DSMRSensorEntityDescription(
+        key="actual_treshold_electricity",
+        translation_key="actual_treshold_electricity",
```

Comment:
Is this spelt correctly? (Threshold vs treshold)

Reply:
Thnk you, I fixed the strings on HA side, but I can't fix the OBIS reference, as it's outside of this repo.